site/content/en/docs/api_sdk/sdk/lowlevel-api.md

```diff
@@ -103,146 +103,122 @@ with ApiClient(configuration) as api_client:
     assert task.size == 4
 ```
 
-## Available API Endpoints
+## ApiClient and configuration
 
-All URIs are relative to _`http://localhost`_
+The starting point in the low-level API is the `cvat_sdk.api_client.ApiClient` class.
+It encapsulates session and connection logic, manages headers and cookies,
+and provides access to different APIs.
 
-APIs can be instanted directly like this:
+To create an instance of `ApiClient`, you need to set up a `cvat_sdk.api_client.Configuration`
+object and pass it to the `ApiClient` class constructor. Additional connection-specific
+options, such as extra headers and cookies can be specified in the class constructor.
+`ApiClient` implements the context manager protocol. Typically, you create `ApiClient` this way:
+
+```python
+from cvat_sdk.api_client import ApiClient, Configuration
+
+configuration = Configuration(host="http://localhost")
+with ApiClient(configuration) as api_client:
+    ...
+```
+
+After creating an `ApiClient` instance, you can send requests to different server endpoints
+via `*_api` member properties and directly, using the `rest_client` member.
+[Read more](#api-wrappers) about API wrappers below.
+
+Typically, the first thing you do with `ApiClient` is login.
+[Read more](#authentication) about authentication options below.
+
+## Authentication
+
+CVAT supports 2 authentication options:
+- basic auth, with your username and password
+- token auth, with your API key
+
+Token auth requires a token, which can be obtained after performing the basic auth.
+
+Low-level API supports 2 ways of authorization.
+You can specify authorization in the `Configuration` object:
+
+```python
+configuration = Configuration(
+    username='YOUR_USERNAME',
+    password='YOUR_PASSWORD',
+)
+```
+
+```python
+configuration = Configuration(
+    api_key={
+        "sessionAuth": "<sessionid cookie value>",
+        "csrfAuth": "<csrftoken cookie value>",
+        "tokenAuth": "Token <auth key value>",
+    }
+)
+```
+
+Or you can perform regular login using the `auth_api` member of `ApiClient` and
+set the `Authorization` header using the `Token` prefix.
+
+```python
+from cvat_sdk.api_client import models
+
+(auth, _) = api_client.auth_api.create_login(
+    models.LoginRequest(username=credentials[0], password=credentials[1])
+)
+
+assert "sessionid" in api_client.cookies
+assert "csrftoken" in api_client.cookies
+api_client.set_default_header("Authorization", "Token " + auth.key)
+```
```

Comment:
When we just put auth params in the config, they are passed with each request. Typically, it's not what we want, and it can even be a security problem in some cases. With this method, we obtain a token and use it instead. A token can be used to compromise data, of course, but it cannot be used for login and to change the password (since the current password is needed). It can be saved and reused across different runs (which we need to implement in CLI tbh). It's just the way we obtain tokens. Another way if to take them from the browser cookies.
I've described my ideas about API tokens in #4961.